modules/integration/tests-integration/tests-backend/src/test/resources/adaptive.bat

```diff
@@ -0,0 +1,164 @@
+rem    NASHORN_VERSION   	OpenJDK Nashorn Version
+rem                   	
+rem    ASM_VERSION   	ASM Util, Commons Version.
+rem
+rem -----------------------------------------------------------------------------
+
+set NASHORN_VERSION=15.3
+set ASM_VERSION=9.2
+
+set SERVER_RESTART_REQUIRED="false"
+
+set CARBON_HOME=%1
+set DISABLE=%2
+
+set LIB_REPO=%CARBON_HOME%\repository\components\lib
+
+setlocal enabledelayedexpansion
+
+rem commandline arguement 'DISABLE' or 'disable' is passed 
+if "%DISABLE%"=="DISABLE" goto disableAdaptiveAuth 
+if "%DISABLE%"=="disable" goto disableAdaptiveAuth 
+rem no commandline arguements are passed
+goto enableAdaptiveAuth
+
+:disableAdaptiveAuth
+set LOCAL_NASHORN_VERSION=""
+set LOCAL_ASM_VERSION=""
+if exist "%CARBON_HOME%\repository\components\lib\nashorn-core-*.jar" (
+  set SERVER_RESTART_REQUIRED="true"
+  for /f "delims=" %%i in ('dir /s /b %CARBON_HOME%\repository\components\lib\nashorn-core-*.jar') do set "location=%%i"
+  for %%j in (!location!!) do set "full_artifact_name=%%~nxj" 
+  for /f "tokens=3 delims=-" %%k in ("!full_artifact_name!") do set "artifact_name=%%k"
+  for /f "tokens=1,2 delims=." %%l in ("!artifact_name!") do set "LOCAL_NASHORN_VERSION=%%l.%%m"
+  echo Remove existing Nashorn library from lib folder: !full_artifact_name!
+  del !location!
+  echo Nashorn library Removed from components\lib.  
+)
+if exist "%CARBON_HOME%\repository\components\dropins\nashorn_core_*.jar" (
+  set SERVER_RESTART_REQUIRED="true"
+  for /f "delims=" %%i in ('dir /s /b %CARBON_HOME%\repository\components\dropins\nashorn_core_!LOCAL_NASHORN_VERSION!*.jar') do set "location=%%i"
+  for %%j in (!location!!) do set "full_artifact_name=%%~nxj" 
+  echo Remove existing Nashorn library from dropins: !full_artifact_name!
+  del /f !location!
+  echo Nashorn library Removed from components\dropins.  
+)
+if exist "%CARBON_HOME%\repository\components\lib\asm-util-*.jar" (
+  set SERVER_RESTART_REQUIRED="true"
+  for /f "delims=" %%i in ('dir /s /b %CARBON_HOME%\repository\components\lib\asm-util-*.jar') do set "location=%%i"
+  for %%j in (!location!!) do set "full_artifact_name=%%~nxj" 
+  for /f "tokens=3 delims=-" %%k in ("!full_artifact_name!") do set "artifact_name=%%k"
+  for /f "tokens=1,2 delims=." %%l in ("!artifact_name!") do set "LOCAL_ASM_VERSION=%%l.%%m"
+  echo Remove existing ASM Util library from lib folder: !full_artifact_name!
+  del !location!
+  echo ASM Util library Removed from components\lib.  
+)
+if exist "%CARBON_HOME%\repository\components\dropins\asm_util_*.jar" (
+  set SERVER_RESTART_REQUIRED="true"
+  for /f "delims=" %%i in ('dir /s /b %CARBON_HOME%\repository\components\dropins\asm_util_!LOCAL_ASM_VERSION!*.jar') do set "location=%%i"
+  for %%j in (!location!!) do set "full_artifact_name=%%~nxj" 
+  echo Remove existing ASM Util library from dropins: !full_artifact_name!
+  del !location!
+  echo ASM Util library Removed from components\dropins.  
+)
```

Comment:
Can't we use a function to delete the jars from specific file paths? Would make the code more modular and easier to reuse.